docs/pouch-express.md

```diff
@@ -0,0 +1,129 @@
+| `databases` (required)   | `{Object}` | An object that describes one or more databases to create.  See below for full details. |
+| `dbOptions`              | `{Object}` | Options that will be used when constructing each individual database. See [the PouchDB docs](https://pouchdb.com/api.html#create_database) for supported options. ]
+| `expressPouchConfig`     | `{Object}` | Options that will be used when constructing the express-pouchdb instance. See [the express-pouchdb docs](https://github.com/pouchdb/express-pouchdb#api) for supported options.|
+| `expressPouchConfigPath` | `{String}` | The path to the temporary file where the settings in `expressPouchConfig` will be stored and read by express-pouchdb.  The settings in `expressPouchConfig` will be saved to a file named `pouchdb.conf` in this directory.  Defaults to `os.tmpdir() + "/pouch.conf"`.|
+
+### The `databases` option
+
+The `databases` option is a hash, keyed by database name.  Each database may optionally contain a `data` element, which
+is a string or an array of strings that represents the path to a JSON file (see below for the formats supported).  A
+path can be the full path to a file on the local machine, or can be a package-relative path, such as `%my-package/tests/data/users.json`.
+
+Here is an example `databases` option that demonstrates all variations:
+
+```
+databases: {
+    fullPath: { data: "/tmp/file.json" },
+    packageRelative: { data: "%my-package/tests/data/file.json" },
+    array: {
+        data: [ "%my-other-package/tests/data/file1.json", "%my-other-package/tests/data/file2.json"]
+    },
+    withCustomOptions: {
+        dbOptions: {
+            autoCompaction: true
+        }
+    },
+    withCustomOptionsAndData: {
+        data: "%my-package/tests/data/file.json",
+        dbOptions: {
+            autoCompaction: true
+        }
+    },
+    empty: {} // Will be created, but without any data
+}
+```
+## Component Invokers
+
+### `{that}.cleanup()`
+* Returns: A `Promise` that will be resolved once cleanup is complete (the `onCleanupComplete` event should also be fired.
+
+This invoker is called when the `onCleanup` event is fired, which should indicate that it is time to remove any existing
+data.  This grade provides only a stub, implementations are expected to override it with their own invoker.
+
+### `{that}.initDbs()`
+* Returns: A `Promise` that will be resolved once all databases have been initialized.
+
+Initialize all of the databases configured in `options.databases` (see above).  The approach used varies depending on
+whether we are working with filesystem or in-memory storage.  The base package provides only a stub which must be
+overriden with another invoker.
+
+### `{that}.middleware(request, response, next)`
+* `request`: The [request object](http://expressjs.com/en/api.html#req) provided by Express, which wraps node's [`http.incomingMessage`](https://nodejs.org/api/http.html#http_class_http_incomingmessage).
```

Comment:
I've commented before on this copypasta language from gpii-express - this isn't an example of "wrapping", it simply is an instance of http.incomingMessage